### DISCUSSION TITLE
How do I get Linux like toolbar with new tab button  on Mac?

### DISCUSSION THREAD

**Discussion by khurshid-alam** [ghostty-org/ghostty#4027] [Q&A] [answered]
I like to have a new tab button on toolbar so that I don't have to use keyboard shortcuts or menubar buttons? How do I enable it ?

**Comment by pluiedev** [comment]
What do you mean by toolbar here?

**Comment by rhodes-b** [comment, marked as answer]
Mac should have a new tab button by default afaik its a plus symbol in the top right corner? (maybe it only shows up when you already have 2 tabs)

**Reply by taylrfnt** [reply]
This is the right answer, you need 2 or more tabs with the native MacOS title bar to show the new tab symbol/area.  Native title bars in MacOS don't show tabs until after 2 tabs are open in the same window

**Reply by khurshid-alam** [reply]
This is a problem. I like to open new tab first time without resorting to keyboard shortcuts.

**Reply by pluiedev** [reply]
We don't really have much control over tabs in macOS because again it's a native feature implemented by macOS itself. This might be possible when we switch to custom tabs, though.

**Reply by khurshid-alam** [reply]
Thanks. Mac can show + button on toolbar even with single tab open. iTerm and Warp all does it.
<img width="710" alt="image" src="https://github.com/user-attachments/assets/ccf0152f-7149-48b1-96ba-74d92112bc5a" />


 I hope this custom tab feature will be implemented soon.
